### PR TITLE
Add unary - for Symmetric

### PR DESCRIPTION
**Pull request by cpk26**
Add unary minus for symmetric matrices

### CODE REVIEW

**Comment by tkelman** [comment]
Thanks, looks good. Add a test?

**Comment by cpk26** [comment]
Sounds good.

**Comment by vtjnash** [comment]
appveyor had an unrelated internal failure (https://ci.appveyor.com/project/StefanKarpinski/julia/build/1.0.11446/job/8bqrhqpy0jrbjxmu)

File: test/linalg/symmetric.jl [hunk 1]
```diff
@@ -209,3 +209,9 @@ let A = [1.0+im 2.0; 2.0 0.0]
     @test !ishermitian(A)
     @test_throws ArgumentError Hermitian(A)
 end
+
+# Unary minus for Symmetric matrices
+let A = Symmetric(randn(5,5))
+    B = -A
+    @test A + B ≈ zeros(5,5)
```
Comment (andreasnoack) [inline test/linalg/symmetric.jl]: Should probably use `==` here.

**Pull request merged**